### DISCUSSION TITLE
Fastlane short_description.txt too long

### DISCUSSION THREAD

**Discussion by IzzySoft** [FossifyOrg/General-Discussion#425] [General]
short description is limited to 80 chars by Fastlane specs, everything behind is cut off. Currently, there are 4 locales where the latter will happen:

* metadata/org.fossify.calendar/es-ES/short_description.txt has 108 chars
* metadata/org.fossify.calendar/eu-ES/short_description.txt has 85 chars
* metadata/org.fossify.calendar/hi-IN/short_description.txt has 93 chars
* metadata/org.fossify.calendar/sv-SE/short_description.txt has 90 chars

and while on it, also 2 in gallery:

* metadata/org.fossify.gallery/pt/short_description.txt has 83 chars
* metadata/org.fossify.gallery/sv-SE/short_description.txt has 103 chars

Can you please check-and-fix? Thanks!

**Comment by naveensingh** [comment]
It seems those strings were translated _before_ the length check was enforced on Weblate. Fixed in https://github.com/FossifyOrg/Calendar/commit/7748d5f81938459ac233c93237cef8402a7ea82a, https://github.com/FossifyOrg/Gallery/commit/d19f8156b9385fe0794cf086c8672c26e97c90b0, and https://github.com/FossifyOrg/File-Manager/commit/d96a14840439f16065515d01de093911238f0dc0

Thanks!

**Comment by IzzySoft** [comment]
Thanks!
